app/helpers/phase_label_helper.rb

```diff
@@ -8,7 +8,7 @@ def render_phase_label(presented_object, message)
   end
 
   def survey_url
-    url = "https://www.smartsurvey.co.uk/s/HGLV5/"
+    url = "https://www.gov.uk/service-manual/communities/contact-the-service-manual-and-service-standard-team"
     url += "?c=#{request.path}" if request.path.present?
     url
   end
```

Comment:
```suggestion
```

I think we can remove needing this function and just specify the URL in the template.